client/src/amplify-ui/block-switcher/block-switcher.tsx

```diff
@@ -49,24 +59,43 @@ export class AmplifyBlockSwitcher {
     }
   }
 
+  handleInitSelectedTabHeading() {
```

Comment:
Could this be just `initSelectedTabHeading()`?

Reply:
Agree. That's a better name.